symbology/simple-fill-symbol/README.md

```diff
@@ -1,37 +1,38 @@
-# Simple Fill Symbol
+# Simple fill symbol
 
-Change a graphic's fill color, outline, and style properties.
+Change a polygon's fill color, outline color, and fill style properties.
```

Comment:
strictly speaking, the symbol is applied to a `Graphic`'s fill colour etc. The graphic is built using the polygon: and therefore a graphic with a polygon geometry is styled.
```suggestion
Change a graphic's fill color, outline color, and fill style properties.
```